### PR TITLE
Cleanups for release

### PR DESCRIPTION
**Pull request by dco-lentz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
 # com.dean.interval-tree
 
-This library provides a colletion of data structures implemented using a
-modular, extensible weight balanaced persistent binary tree:
+This library provides a collection of data structures implemented using a
+modular, extensible, foldable, weight balanaced persistent binary tree:
```
Comment (alexandergunnarson) [inline README.md:4]: "balanaced" → "balanced"?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,18 @@
 # com.dean.interval-tree
 
-This library provides a colletion of data structures implemented using a
-modular, extensible weight balanaced persistent binary tree:
```
Comment (alexandergunnarson) [inline README.md:4]: "balanaced" → "balanced"?

**Review by alexandergunnarson — APPROVED** [review]
Nice job! 🙂 Seems like the docs were autogenned from Codox — is there a way to have that build as a step in a GitHub actions flow and then have the root of the docs be available on https://github.io/dco-dev/interval-tree ?

File: README.md (hunk 2 above)
Comment (dco-lentz) [reply README.md:4]: Nice catch!

**Comment by dco-lentz** [comment]
Good idea I’ll look into getting that automated.  I also feel like I should say more in terms of general documentation.

File: README.md (hunk 2 above)
Comment (alexandergunnarson) [reply README.md:4]: Thanks!

**Comment by alexandergunnarson** [comment]
Awesome! And yeah I mean most everyone will know how to make use of ordered maps/sets (though they won't know how this lib compares to e.g. `org.flatland/ordered` or `frankiesardo/linked`), but I suspect few people will know how to make use of interval maps/sets (or how this lib compares to e.g. `org.dthume/data.interval-treeset`). So I think it might be nice to make a brief comparison to the broader landscape, as well as document basic usage (with further documentation to be found in Codox). Shouldn't be longer than like 50 LOC of Markdown, probably?

File: README.md (hunk 2 above)
Comment (dco-lentz) [reply README.md:4]: resolved by https://github.com/dco-dev/interval-tree/pull/6/commits/f2b97dc1266632b350894efa1bde1e21eaa1f6e7

**Comment by alexandergunnarson** [comment]
I think this is looking great so far — love the additions and your usual Dan ASCII art! 😄

**Comment by dco-lentz** [comment]
Thanks!  Do you happen to have an example of a repo with the automated doc build done well?  I was looking at how this might be done with a push inside of a workflow, but was wondering if that might be more of a nuisance than just building them manually before release

**Comment by alexandergunnarson** [comment]
@dco-lentz Sorry for the delay — my family and I just moved to LA and have been getting situated 😅 No worries about auto-building the docs; I think that todo can be left for another time. If building manually is easier, so be it :) Unfortunately I don't have an example offhand of auto-building, but I know the process exists!

**Comment by dco-lentz** [comment]
Best of luck in your new home!

**Comment by alexandergunnarson** [comment]
Thanks @dco-lentz!! Looking forward to this being in Clojars 😃

**Pull request merged**